runtime/polkadot/src/weights/pallet_staking.rs

```diff
@@ -221,18 +219,18 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	// Storage: Staking Payee (r:0 w:1)
 	// Storage: Staking SpanSlash (r:0 w:2)
 	fn force_unstake(s: u32, ) -> Weight {
-		(47_441_000 as Weight)
+		(50_620_000 as Weight)
 			// Standard Error: 1_000
-			.saturating_add((679_000 as Weight).saturating_mul(s as Weight))
+			.saturating_add((723_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(11 as Weight))
 			.saturating_add(T::DbWeight::get().writes(12 as Weight))
 			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(s as Weight)))
 	}
 	// Storage: Staking UnappliedSlashes (r:1 w:1)
 	fn cancel_deferred_slash(s: u32, ) -> Weight {
-		(976_712_000 as Weight)
-			// Standard Error: 56_000
-			.saturating_add((4_963_000 as Weight).saturating_mul(s as Weight))
+		(3_348_741_000 as Weight)
+			// Standard Error: 223_000
```

Comment:
Does it play any role if this number raises so significantly?

Reply:
It would imply that there was a change that was not re-benchmarked, as in general this should not happen.